internal/cmd/gocdk/init.go

```diff
@@ -87,6 +88,12 @@ func doInit(ctx context.Context, pctx *processContext, dir, modpath string, allo
 	if err := static.Do(projectDir, nil, actions...); err != nil {
 		return xerrors.Errorf("gocdk init: %w", err)
 	}
+	// Add a "dev" biome with a local launcher, using a new pctx that discards
+	// the normal output of "gocdk biome add".
+	subpctx := newProcessContext(projectDir, pctx.stdin, ioutil.Discard, ioutil.Discard)
+	if err := biomeAdd(ctx, subpctx, "dev", "local"); err != nil {
```

Comment:
Is there a way we can factor out the code in biome add such that we don't have to create a new process context?

Reply:
Yes, but not easily; it uses a bunch of `pctx` fields so we'd end up needing to pass almost all of them. Not sure that's an improvement.

Note that the working directory of this `subpctx` is also different than the base one.

Reply:
Mind opening an issue? It feels like there should be some way of mitigating this, but I don't have a concrete suggestion right now.

Reply:
https://github.com/google/go-cloud/issues/2348